Dockerfile

```diff
@@ -16,18 +16,22 @@ COPY controllers/ controllers/
 COPY converters/ converters/
 COPY constants/ constants/
 COPY builder/ builder/
+COPY platform/ platform/
 COPY utils/ utils/
+COPY install/ install/
+COPY resources/ resources/
 
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-#FROM gcr.io/distroless/static:nonroot
-FROM registry.access.redhat.com/ubi8/ubi-micro:latest
+FROM gcr.io/distroless/static:nonroot
```

Comment:
yes, for the productized version it will be better to use the ubi-micro one.

For the community one I think that we can use the distroless that it is the default one proposed by the operator sdk.

What do you think?

Reply:
can be :)